ocs_ci/deployment/deployment.py

```diff
@@ -646,6 +649,109 @@ def patch_default_sc_to_non_default(self):
                 f"--request-timeout=120s"
             )
 
+    def uninstall_osc(self):
+        ocp_obj = ocp.OCP()
+        provisioners = ['openshift-storage.rbd.csi.ceph.com',
+                        'openshift-storage.cephfs.csi.ceph.com',
+                        'openshift-storage.noobaa.io/obc']
+
+        # STEP 1:List the storage classes
+        sc_list = get_all_storageclass_names(all_ns=True)
+        for storage_class in sc_list:
+            sc = ocp.OCP(kind=constants.STORAGECLASS, resource_name=storage_class)
+            pro = sc.get('provisioner')
+            if pro != provisioners[0] and pro != provisioners[1] and pro != provisioners[2]:
```

Comment:
I suggest you use any() instead:
```python
if any(sc.get('provisioner') in provisioners for sc in sc_list):
    sc_list.remove(storage_class)
```

Reply:
tried to implement this ``` if any(sc.get('provisioner') in provisioners for sc in sc_list):
    sc_list.remove(sc) ``` and it doesn't work because ` sc` declared inside the any part and the remove func doesn't see it.